stock_move_compute_move_lines/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2012-2014 Alexandre Fayolle, Camptocamp SA
```

Comment:
@BorisFL This module seems to not be part of stock_picking_batch_extended migration

Reply:
In a previous [comment](https://github.com/OCA/stock-logistics-workflow/pull/1236#discussion_r1180394498), you asked me to split that functionality into another module. Should I directly delete that module in this commit?

Reply:
Please... remove this module from this commit